lib/Rx/Observable/TimerObservable.php

```diff
@@ -25,11 +26,10 @@ public function __construct($dueTime, SchedulerInterface $scheduler = null)
         $this->scheduler = $scheduler;
     }
 
-    public function subscribe(ObserverInterface $observer, SchedulerInterface $scheduler = null)
+    public function subscribe(ObserverInterface $observer)
```

Comment:
The scheduler should already be decided by the observable already, not on subscription, I would have thought?